pygmt/base_plotting.py

```diff
@@ -1167,8 +1167,10 @@ def basemap(self, **kwargs):
 
         """
         kwargs = self._preprocess(**kwargs)
-        if not ("B" in kwargs or "L" in kwargs or "T" in kwargs):
-            raise GMTInvalidInput("At least one of B, L, or T must be specified.")
+        if not ("B" in kwargs or "L" in kwargs or "Td" in kwargs or "Tm" in kwargs):
+            raise GMTInvalidInput(
+                "Frame, map_scale, compass, or rose must be specified."
```

Comment:
The capitalized word "Frame" is confusing.

Reply:
Just pushed a commit with the change